README.md

```diff
@@ -30,53 +31,75 @@ For more information:
 
 > `npm install --save fluture`
 
+### In ES5 or older environments
+
+Fluture depends on these functions being present: `Object.create`,
+`Object.assign` and `Array.isArray`. You may need to polyfill one or more.
+
 ```js
-const fs = require('fs');
-const Future = require('fluture');
+var fs = require('fs');
+var Future = require('fluture');
 
-const getPackageName = file =>
-  Future.node(done => fs.readFile(file, 'utf8', done))
+var getPackageName = function(file){
+  return Future.node(function(done){ fs.readFile(file, 'utf8', done) })
```

Comment:
My preference is not to have semicolons it preceding `}`. If you feel strongly about it you could open a separate issue to change it across the code-base. Now that eslint-plugin-markdown is installed, changing the eslint rule will also affect the README. :)